.github/workflows/test.yml

```diff
@@ -63,6 +68,10 @@ jobs:
     steps:
       - uses: actions/checkout@main
       - uses: Swatinem/rust-cache@v2
+      - name: Install LLVM/Clang
+        uses: KyleMayes/install-llvm-action@v2
```

Comment:
Is this the "canonical" way to install LLVM? The action [repo](https://github.com/kylemayes/install-llvm-action) doesn't seem to be super popular - but maybe that's fine?